src/palace/manager/api/axis.py

```diff
@@ -211,7 +211,9 @@ def label(cls) -> str:
     def description(cls) -> str:
         return ""
 
-    def __init__(self, _db: Session, collection: Collection) -> None:
+    def __init__(
```

Comment:
Allow a bearer token to be reused for cases when user is making multiple calls to the same collection but in different transactions in rapid succession.  This reuse dramatically reduces the number of calls that need to be made to the axis token endpoint, especially when there are many api calls being processed in parallel